{{cookiecutter.src_dirname}}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0  # Update as necessary with "pre-commit autoupdate"
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+      - id: debug-statements
+  - repo: https://github.com/psf/black
+    rev: 21.12b0  # Update as necessary with "pre-commit autoupdate"
+    hooks:
+      - id: black
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
+# Use local for hooks not installed automatically by pre-commit
+  - repo: local
+    -   id: flake8
+        name: flake8
```

Comment:
We could use (https://flake8.pycqa.org/en/latest/user/using-hooks.html), but:
1. We would need to add our flake8_level12 repo as a dependency since pre-commit installs to an isolated virtualenv
2. Editors which pick up on the installed version of flake8 would still need flake8 installed in the project virtualenv
3. If we use the flake8 hook, we could remove the flake8 section in tox, since the pre-commit section in tox would now run flake8